packages/web-app/src/modules/reward-views/components/RewardHeaderBar.tsx

```diff
@@ -111,6 +111,7 @@ class _RewardHeaderBar extends Component<Props> {
   render() {
     const { reward, classes, ...rest } = this.props
 
+    let donation = reward?.tags.includes('donations')
```

Comment:
The tags are not always the same case. Instead we should match against the `category` which will remain consistent:

```suggestion
    let donation = reward?.category === "donation"
```